tests/actions/ReportTest.ts

```diff
@@ -530,7 +533,6 @@ describe('actions/Report', () => {
                         value: {
                             1: REPORT_ACTION,
                         },
-                        shouldNotify: true,
```

Comment:
WHy this change?

Reply:
reverted with the comment as it was done in another test